tools/Google.Cloud.Tools.Analyzers.Tests/Google.Cloud.Tools.Analyzers.Tests.csproj

```diff
@@ -0,0 +1,183 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Well as I said above, it caused the tests to fail to load System.Collections.Immutable, but I played around with it a bit more just now and was able to get things working by removing the binding redirects to the Roslyn libraries which are added by the analyzer template.